src/services/services.tsx

```diff
@@ -0,0 +1,12 @@
+const getData = async () => {
+  await fetch("https://jsonplaceholder.typicode.com/posts/1")
```

Comment:
I am not sure that here asyn / await is needed as you already use fetch().then 
Choose 1 variant.

Example: https://dev.to/masteringjs/using-then-vs-async-await-in-javascript-2pma#:~:text=Async%2Fawait%20and%20then(),callback%20until%20the%20promise%20settles.

Reply:
Please, use `async/await` variant - it's current modern style